subhub/hub/stripe/customer.py

```diff
@@ -98,13 +65,44 @@ def run(self):
 class StripeCustomerSubscriptionCreated(AbstractStripeHubEvent):
     def run(self):
         logger.info("customer subscription created", payload=self.payload)
+        invoice_number = None
+        invoice_id = None
+        charge = None
+        last4 = None
+        brand = None
         try:
             customer_id = self.payload.data.object.customer
             updated_customer = stripe.Customer.retrieve(id=customer_id)
             user_id = updated_customer.metadata.get("userid")
         except InvalidRequestError as e:
             logger.error("Unable to find customer", error=e)
             raise InvalidRequestError(message="Unable to find customer", param=str(e))
+        try:
+            logger.info("invoice", latest=self.payload.data.object.latest_invoice)
+            invoice_id = self.payload.data.object.latest_invoice
+            latest_invoice = stripe.Invoice.retrieve(id=invoice_id)
+            logger.info(
+                "latest invoice",
+                latest_invoice=latest_invoice.number,
+                charge=latest_invoice.charge,
+            )
+            invoice_number = latest_invoice.number
+            charge = latest_invoice.charge
+        except InvalidRequestError as e:
+            logger.error("Unable to retrieve invoice", error=e)
+            raise InvalidRequestError(
+                message="Unable to retrieve invoice", param=str(e)
```

Comment:
don't do this; instead use the `raise InvalidRequestError(blah) from e` pattern.  We spent a lot of time undoing stringifying exception instances.